File: Leetcode/2037/2037.cpp
class Solution {
public:
    int minMovesToSeat(vector<int>& seats, vector<int>& students) {
        sort(seats.begin(),seats.end());
        sort(students.begin(),students.end());
        int n = (int)seats.size();
        int counter = 0;
        for (int i = 0; i < n; ++i) {
            counter += abs(seats[i] - students[i]);
        }
        return counter;
    }
};

File: Leetcode/2037/2037.go
func abs(x int) int {
	if x < 0 {
		x = -x
	}
	return x
}

func minMovesToSeat(seats []int, students []int) int {
	sort.Ints(seats)
	sort.Ints(students)
	ans := 0
	for i := 0; i < len(students); i++ {
		ans += abs(seats[i] - students[i])
	}
	return ans
}